packages/wonder-blocks-modal/components/two-column-modal/__snapshots__/two-column-modal.test.js.snap

```diff
@@ -3,13 +3,16 @@
 exports[`TwoColumnModal mobile with footer should match snapshot 1`] = `
 <ModalDialog
   style={
-    Object {
-      "_definition": Object {
-        "background": "#ffffff",
+    Array [
+      Object {
+        "_definition": Object {
+          "background": "#ffffff",
+        },
+        "_len": 24,
+        "_name": "smallDialog_144oaus",
       },
-      "_len": 24,
-      "_name": "smallDialog_144oaus",
-    }
+      undefined,
+    ]
```

Comment:
I'm not sure why we aren't printing out the style objects for these tests but that's something the Frontend Infra team can investigate independently of this PR.